Formula/ooni-cli.rb

```diff
@@ -0,0 +1,43 @@
+class OoniCli < Formula
+  desc "Next generation OONI Probe CLI"
+  homepage "https://ooni.org/install"
+  url "https://github.com/ooni/probe-cli/archive/v3.0.7.tar.gz"
+  sha256 "01e1e1a8d7d8fff04a8c098cfd33c84c507cc6efb88c597714b4917b9d0d64e1"
+  license "BSD-3-Clause"
+
+  depends_on "go" => :build
+
+  conflicts_with "ooniprobe", because: "old version of the client, installs binary with the same name"
+
+  def install
+    system "go", "build", "-ldflags",
+            "-s -w", "-tags", "DISABLE_QUIC",
+            "./cmd/ooniprobe"
+    bin.install "ooniprobe" => "ooniprobe"
```

Comment:
```suggestion
    system "go", "build", *std_go_args, "-ldflags",
            "-s -w", "-tags", "DISABLE_QUIC",
            "./cmd/ooniprobe"
```

Reply:
Your suggestion does not quite work because `*std_go_args` sets the output file name to the formula name, which is not correct in this case. But I changed the output to `#{bin}/ooniprobe` which allows for the removal of call to `bin.install`.